src/components/Card.tsx

```diff
@@ -23,17 +30,31 @@ const Card: React.FC<CardProps> = ({
     };
 
     return (
-        <div className="card-container" id={id}>
-            <div className="card-item">Stadium - {stadiumName}</div>
-            <div className="card-item">City - {city}</div>
-            <div className="card-item">Capacity - {capacity}</div>
-            <div className="card-item">Field - {fieldType}</div>
-            <div className="form-buttons card-button">
-                <Button type="button" onClick={handleRemove}>
+        <MiuCard id={id} className="card-container" sx={{padding: "5px"}}>
```

Comment:
Try not to use hardcoded values like `padding: "5px"` You can use spacing from the theme. By default, if you set `padding: 1` it`s the same as `padding: 8px`, but you can change this` 8px` factor in your theme. Also you can write just `p `instead of `padding`, `pt `instead of `padding-top` etc.